### DISCUSSION TITLE
Feature Request: Still image from the last video frame if video is muted.

### DISCUSSION THREAD

**Discussion by misi** [edumeet/edumeet#847] [Ideas]
Meg lehetne oldani, hogy egy résztvevő, ha kikapcsolja a webkameráját, akkor ne egy szürke avatar maradjon a helyén, hanem esetleg egy kép. Pl.: lehetne az utolsó frame amit még a codec feldolgozott?

**Comment by Alex-YddeR** [comment]
This will make users that video got hanged. instead out the profile image

**Comment by so010** [comment]
Alternatively: don't show participant without video at all and 
if one of non-video participants is speaker: show a centered overlay "{displayName} is talking"

**Comment by Astagor** [comment]
I also think that showing the last frame is not a good idea for privacy issues. If somebody decided to turn cam off, he should not be seen anymore.

**Comment by misi** [comment]
Ok so let use the profile image. :)
